package.json

```
@@ -25,7 +25,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
-    "@theforeman/builder": "^10.1.0",
+    "@theforeman/builder": ">= 12.0.1",
```

Thanks, @ehelms, but what I meant is to update all others as well (I think they should be synced?..)

```suggestion
    "@theforeman/builder": ">= 12.0.1",
    "@theforeman/eslint-plugin-foreman": ">= 12.0.1",
    "@theforeman/vendor-dev": ">= 12.0.1",
    "@theforeman/test": ">= 12.0.1",
    "@theforeman/stories": ">= 12.0.1",
```

---

Done.